mix.exs

```diff
@@ -111,7 +111,9 @@ defmodule Sentry.Mixfile do
       # Required by Phoenix.LiveView's testing
       {:floki, ">= 0.30.0", only: :test},
       {:oban, "~> 2.17 and >= 2.17.6", only: [:test]},
-      {:quantum, "~> 3.0", only: [:test]}
+      {:quantum, "~> 3.0", only: [:test]},
+      {:opentelemetry, "~> 1.4"},
+      {:opentelemetry_api, "~> 1.3"}
```

Comment:
Are we ok with Sentry *always* depending on OTel libraries, even when users won't care about transactions?

Reply:
@whatyouhide right, this could become optional